### DISCUSSION TITLE
Hidden Project Files

### DISCUSSION THREAD

**Discussion by Microphunk** [directus/directus#4119] [Feature Requests]
Would be nice, if the Project logo and project background+foreground image would be hidden from File Directory to users, since those are system/app pictures.

**Comment by benhaynes** [comment]
Oh, that's an interesting idea. Perhaps we have a default "System" virtual folder where these system files are organized... maybe we can set default role permissions to not READ that folder (but still READ the files within)?

**Reply by Microphunk** [reply]
Forgot to mention, that User profile avatars could also be made invisible in File Directory.

**Reply by rijkvanzanten** [reply]
> Forgot to mention, that User profile avatars could also be made invisible in File Directory.

This does get a little opinionated though.. I see how it can be useful, but on the other hand it's not unthinkable you'd re-use these same avatars on like a "Team" page on your website 🤔  I don't think we can "just" hide them globally based on use in directus_* contexts

**Reply by benhaynes** [reply]
I think what we _can_ do is have a few default virtual folders for organizing these pseudo-system assets. By adding a `User Avatars` folder and uploading all applicable files there, it makes it easier for admins to update permissions to filter them out. Same thing for the Project/Public Media in Settings.

**Reply by ghost** [reply]
Another idea - have a designated /public folder, /users/{id} folders only accessible by the specific user, or /roles/{id} folders only accessible by specific roles. This can be achieved now with the current field permission system, but not in an automated way.

The clearest existing way of organizing files is using the multiple files field/interface - since that clearly shows the user which files are related to what, but unfortunately doesn't support downloading files yet (https://github.com/directus/directus/issues/3927).

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
